BookStore/src/main/java/controller/BookUpdateController.java

```diff
@@ -0,0 +1,47 @@
+package controller;
+
+import entities.Book;
+import service.BookService;
+import service.BookServiceImpl;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/BookUpdate")
+public class BookUpdateController extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    BookService bookService = new BookServiceImpl();
+    String nameBook = "";
+    String nameAuthor = "";
+    Integer idBook = 0;
```

Comment:
Why do you declare these variables as global?

Reply:
idBook em khai báo global để khi get được từ hàm doGet thì sử dụng lại chính id đó ở hàm doPost để cập nhật lại sách đó